medusa/clients/torrent/qbittorrent_client.py

```diff
@@ -79,9 +79,10 @@ def _add_torrent_uri(self, result):
     def _add_torrent_file(self, result):
 
         self.url = '{host}command/upload'.format(host=self.host)
+        # Using a mock_filename.torrent, as it doesn't have any purpose, besides that the client requests for it.
         files = {
             'torrents': (
-                '{result}.torrent'.format(result=result.name),
+                'mock_filename.torrent',
```

Comment:
I would just remove it, as it doesn't seem to be needed at all?

Reply:
I can't test. But @DarkSupremo mentioned you can't discard it.
`In all tests, looks that this is being completly discarted by qbittorrent.. You can't just remove that because it will complain, but an fake name seens to do the job just fine and fix the problem..`

Reply:
Actually, i did more tests and i probably messed with it earlier, you can completly get rid of that and it will work...

Since it's just one torrent sent by time, you can also change to this, that will also work:
```
files = {'torrents': result.content}
```

Reply:
This endpoint is meant for uploading a torrent file https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-Documentation#upload-torrent-from-disk
Isn't this used for blackhole?

Reply:
Actually, as i understood, it can be sent as POST or as entirely in the 'torrents' body as raw data:
```
Property | Type | Description
torrents | raw | Raw data of torrent file. torrents can be presented multiple times.
```
There's also an implementation of that here, that does similar to what i said:
http://python-qbittorrent.readthedocs.io/en/latest/_modules/qbittorrent/client.html
```torrent_files = {'torrents': file_buffer}```

Reply:
My initial comment was a lot bigger at first, but what it comes down to is this:
I can confirm the issue, and that it **does** work without the filename:
```py
files = {'torrents': result.content}
```

1. The reason for why it doesn't work is `requests` encodes the utf-8 encoded filename in the request:
    ```http
    Content-Disposition: form-data; name="torrents"; filename*=utf-8''%D1%84%D0%B0%D0%B9
    ```
    I'm assuming qBittorrent doesn't know how to handle that.

2. Yes, you can drop the `filename` completely.
    If you do that, `requests` automatically sets the filename to the key (`torrents`).
    Just note that _if_ the binary data sent to qBittorrent can't be parsed as a torrent file, this error will be in the response:
    `Error: 'filename' is not a valid torrent file.`
    Which is fine, I guess. The response is exactly the same, thanks to `requests`.